generators/app/templates/src/utils/navUtils.js

```diff
@@ -0,0 +1,10 @@
+export const getCurrentRouteName = navigationState => {
```

Comment:
Isn't this part of another PR ? 🤔

Reply:
Mmmm yes, but it's needed in this PR also. When the one who merged first, the other will update and these changes will disappear